modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/DataEngineFormBuilder.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {Context as ModalContext} from '@clayui/modal';
+import {DragLayer} from 'data-engine-taglib';
+import {Pages, useForm, useFormState} from 'dynamic-data-mapping-form-renderer';
+import {EVENT_TYPES as CORE_EVENT_TYPES} from 'dynamic-data-mapping-form-renderer/js/core/actions/eventTypes.es';
+import fieldDelete from 'dynamic-data-mapping-form-renderer/js/core/thunks/fieldDelete.es';
+import React, {useContext} from 'react';
+
+const DataEngineFormBuilder = () => {
+	const dispatch = useForm();
+	const [{onClose}, modalDispatch] = useContext(ModalContext);
+
+	const {rules} = useFormState();
+
+	return (
+		<div className="data-engine-form-builder ddm-form-builder pb-5">
+			<div className="sheet">
```

Comment:
We will start to encourage the use of Clay here.

```suggestion
			<ClayLayout.Sheet>
```

Reply:
Makes sense! Done!